test/distributed/_composable/fsdp/test_fully_shard_logging.py

```diff
@@ -0,0 +1,61 @@
+# Owner(s): ["module: fsdp"]
+import functools
+import os
+import unittest.mock
+
+import torch.distributed as dist
+from torch._dynamo.test_case import run_tests
+from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
+from torch.testing._internal.inductor_utils import HAS_CUDA
+from torch.testing._internal.logging_utils import LoggingTestCase
+
+requires_cuda = unittest.skipUnless(HAS_CUDA, "requires cuda")
+requires_distributed = functools.partial(
+    unittest.skipIf, not dist.is_available(), "requires distributed"
+)
+
+
+@skip_if_lt_x_gpu(2)
+class LoggingTests(LoggingTestCase):
+    @requires_distributed()
+    def test_fsdp_logging(self):
+        env = dict(os.environ)
+        env["TORCH_LOGS"] = "fsdp"
```

Comment:
Can you explain / document in the PR description when to use `TORCH_LOGS="fsdp"` vs. `TORCH_LOGS="+fsdp"`?

The PR title suggests `"+fsdp"`, but the unit test here is using just `"fsdp"` it looks like.

Reply:
Also, do you know if there is a difference between `TORCH_LOGS="fsdp"` vs. `TORCH_LOGS=fsdp`?

Reply:
The input for the TORCH_LOGS should be a string as `TORCH_LOGS="fsdp"`, but we find that input as `TORCH_LOGS=fsdp` also works

Reply:
In the test here, we test the output stderr here with the expected print logs.
Thanks to @mlazos , [documents](https://docs.python.org/3/library/logging.handlers.html#) here show that stderr had all level log information, StreamHandler prints to `stderr` by default. something else is changing the logging settings later. So that the input here for env["TORCH_LOGS"] makes no difference among "`fsdp", "+fsdp" and "-fsdp"` as `stderr` is the same.